projects/demo/src/modules/app/app.template.html

```diff
@@ -12,22 +12,40 @@
                 tuiLink
                 tuiMode="onLight"
                 href="https://github.com/tinkoff/taiga-ui"
+                title="Taiga UI source code on GitHub"
+                iconAlign="left"
                 rel="noreferrer"
+                icon="assets/images/github.svg"
                 target="_blank"
-                class="app-link"
+                class="link"
             >
-                <img
-                    src="assets/images/github.svg"
-                    alt=""
-                    class="app-github"
-                />
                 GitHub
             </a>
+            <a
+                tuiLink
+                href="https://discord.gg/zrB2EdJjEy"
+                title="Get help on Angular Discord"
+                iconAlign="left"
+                rel="noreferrer"
+                icon="assets/icons/discord.svg"
+                target="_blank"
+                class="link"
+            >
+                Discord
```

Comment:
Let's make them to use the similar colors?
<img width="470" alt="blue" src="https://user-images.githubusercontent.com/35179038/217868905-c63e1d91-20e5-45d0-bbcf-846fdec73a9f.png">

Reply:
for example
<img width="470" alt="black" src="https://user-images.githubusercontent.com/35179038/217869017-9a690f4f-4c45-4e98-b509-538318329849.png">